### ISSUE TITLE
Feature suggestion: differentiation between humanitarian and regular course exams

### ISSUE THREAD

**Issue by gelbermann** [michael-maltsev/cheese-fork#10]
Hi,
Right now, exams for humanitarian courses only appear for their second attempt (Moed B), and since their dates usually coincide with late first attempts for regular courses, the result is a startling time gap between Moed B exams. For example:
![image](https://user-images.githubusercontent.com/41343437/59979478-21e7b100-95f1-11e9-9835-92b18f38757d.png)

Maybe it's possible to display humanitarian exams in a different row? Just a suggestion.

**Comment by michael-maltsev** [comment]
I agree that it could be more convenient to not have it in the second row. On the other hand, I don't think that it deserves a row of its own. 

Another option is to put it in the first row, append an asterisk to it, and say "מועד ב'" on hover. Might work, but not ideal too since it's in the "מועדי א'" row while being a "מועד ב'".

Let's see if anyone has a better suggestion.
